iwyu_output.cc

```diff
@@ -605,6 +636,28 @@ void IwyuFileInfo::ReportForwardDeclareUse(SourceLocation use_loc,
   LogSymbolUse("Marked fwd-decl use of decl", symbol_uses_.back());
 }
 
+void IwyuFileInfo::ReportUsingDeclUse(SourceLocation use_loc,
+                                      const UsingDecl* using_decl,
+                                      const NamedDecl* target_decl,
+                                      bool in_cxx_method_body,
+                                      const char* comment) {  
+  for(auto&& saved_decl : using_decl_lines_) {
+    if(saved_decl.matches(using_decl)) {
+      saved_decl.set_referenced();
+      break;
+    }
+  }
+
+  ReportFullSymbolUse(use_loc, using_decl, in_cxx_method_body, comment);
+  ReportForwardDeclareUse(using_decl->getSourceRange().getBegin(), 
```

Comment:
We don't know it's sufficient, but we also don't know it's not. There's logic later on to convert all functions to FullSymbolUse. This is the same as how it worked previously but now we're running the report through the using decl.